pkg/meta/server.go

```diff
@@ -138,6 +139,15 @@ func (s *Server) Run(ctx context.Context) error {
 func (s *Server) constructHandler() *chi.Mux {
 	router := chi.NewRouter()
 	router.Use(middleware.Timeout(defaultMaxRequestDuration), util.SetServerHeader)
+
+	router.Use(func(next http.Handler) http.Handler {
```

Comment:
Make it a helper function like `util.SetServerHeader` in `util` package?

Reply:
And then later as a separate PR we can move `writeJson()` and `errorRenderer()` there too.